spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -188,12 +234,14 @@ def has_many_children(options = {})
       if options[:presence]
         validates_presence_of :children
       end
-    end.new
+    end
+    .new
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.